package.json

```diff
@@ -33,6 +37,12 @@
     "webpack-cli": "^3.1.1"
   },
   "dependencies": {
+    "body-parser": "^1.18.3",
+    "lodash.pick": "^4.4.0",
+    "lodash.sample": "^4.2.1",
+    "express": "^4.16.3",
```

Comment:
Should be dev dep

Reply:
As above, eslint says it should be dep.

Reply:
like above 100%

Reply:
[https://medium.com/@stalonadsl948/dependencies-vs-devdependencies-926e096a3dee](url)

> Dependencies should contain libs and framewors your app is built on, such as Vue, React, Angular, Express, JQuery and etc. You will agree with me, if I say, that your project wont work without these packages(if you are using them, of course).

Reply:
Just leave it as it is